teacher.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(specialization)
+    super()
+    @specialization = specialization
+  end
```

Comment:
- [ ] Same issue here. Kindly fix your initialize method parameters and the super 👍